api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -87,6 +88,48 @@ message KubernetesPodData {
   repeated string ip_addresses = 1;
 }
 
+message KubernetesNetworkPolicyData {
+  // Specification of the NetworkPolicy.
+  repeated string policy_types = 1; // E.g., "Ingress", "Egress"
+  PodSelector pod_selector = 2;
+  repeated NetworkPolicyRule ingress_rules = 3;
+  repeated NetworkPolicyRule egress_rules = 4;
+
+  // PodSelector defines the pods targeted by the policy.
+  message PodSelector {
+    map<string, string> match_labels = 1;
```

Comment:
You must also support `match_expressions`.
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#labelselector-v1-meta